docs_src/microservices/application/BuiltIn.md

```diff
@@ -391,21 +391,26 @@ There is one response data function included in the SDK that can be added to you
 
 There is one Tags transform included in the SDK that can be added to your pipeline. 
 
-| Factory Method                       | Description                                                  |
-| ------------------------------------ | ------------------------------------------------------------ |
-| NewTags(tags map[string]string) Tags | This factory function returns a `Tags` instance initialized with the passed in collection of tag key/value pairs. This `Tags` instance is used to access the following Tags function that will use the specified collection of tag key/value pairs. |
+| Factory Method                                   | Description                                                  |
+| ------------------------------------------------ | ------------------------------------------------------------ |
+| NewGenericTags(tags map[string]interface{}) Tags | This factory function returns a `Tags` instance initialized with the passed in collection of generic tag key/value pairs. This `Tags` instance is used to access the following Tags function that will use the specified collection of tag key/value pairs. This allows for generic complex complex values for the Tag values. |
```

Comment:
should we put an edgy note of new thing for this?

Reply:
Added.